Tests/Twig/GlobalVariablesTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\Twig;
+
+use Sonata\AdminBundle\Twig\GlobalVariables;
+
+/**
+ * @author Ahmet Akbana <ahmetakbana@gmail.com>
+ */
+final class GlobalVariablesTest extends \PHPUnit_Framework_TestCase
```

Comment:
Thanks for updating, I will keep this in mind. But in this case there won't be any harm of being a `final class`, right?